examples/Extensions/AddGoogleMyBusinessLocationExtensions.php

```diff
@@ -0,0 +1,252 @@
+                    ])
+                ])],
+                'function_string' => new StringValue(['value' => 'IDENTITY(true)']),
+                'operator' => MatchingFunctionOperator::IDENTITY
+            ])
+        ]);
+        // Creates a customer feed operation.
+        $customerFeedOperation = new CustomerFeedOperation();
+        $customerFeedOperation->setCreate($customerFeed);
+
+        // After the completion of the feed ADD operation above the added feed will not be available
+        // for usage in a customer feed until the sync between the Google Ads and GMB accounts
+        // completes. The loop below will retry adding the customer feed up to ten times with an
+        // exponential back-off policy.
+        $numberOfAttempts = 0;
+        $addedCustomerFeed = null;
+        $customerFeedServiceClient = $googleAdsClient->getCustomerFeedServiceClient();
+        do {
+            $numberOfAttempts++;
+            try {
+                // Issues a mutate request to add a customer feed and print its information if the
+                // request succeeded.
+                $addedCustomerFeed = $customerFeedServiceClient->mutateCustomerFeeds(
+                    $customerId,
+                    [$customerFeedOperation]
+                );
+                printf(
+                    "Customer feed created with resource name: '%s'.%s",
+                    $addedCustomerFeed->getResults()[0]->getResourceName(),
+                    PHP_EOL
+                );
+            } catch (GoogleAdsException $googleAdsException) {
+                // Waits using exponential backoff policy.
+                $sleepSeconds = self::POLL_FREQUENCY_SECONDS * pow(2, $numberOfAttempts);
+                // Exits the loop early if $sleepSeconds grows too large in the event that
+                // MAX_CUSTOMER_FEED_ADD_ATTEMPTS is set too high.
+                if ($sleepSeconds > self::POLL_FREQUENCY_SECONDS
+                        * pow(2, self::MAX_CUSTOMER_FEED_ADD_ATTEMPTS)) {
+                    break;
+                }
+                printf(
+                    "Attempt #%d to add the customer feed was not successful."
+                    . " Waiting %d seconds before trying again.%s",
+                    $numberOfAttempts,
+                    $sleepSeconds,
+                    PHP_EOL
+                );
+                sleep($sleepSeconds);
+            }
+        } while ($numberOfAttempts < self::MAX_CUSTOMER_FEED_ADD_ATTEMPTS
+            && is_null($addedCustomerFeed));
+
+        if (is_null($addedCustomerFeed)) {
+            throw new \RuntimeException(
+                'Could not create the customer feed after ' . self::MAX_CUSTOMER_FEED_ADD_ATTEMPTS
+                . ' attempts. Please retry the customer feed ADD operation later.'
+            );
+        }
+        // OPTIONAL: Create a campaign feed to specify which feed items to use at the campaign
```

Comment:
Will these be implemented or are we leaving these as they are now?

Reply:
Leave as is. This is aligned with the golden CL, and it's already a long code example.

Reply:
Ok.